unitary/examples/quantum_rpg/classes.py

```diff
@@ -29,8 +31,9 @@ class Analyst(qaracter.Qaracter):
     def actions(self) -> Dict[str, Callable]:
         return {
             's':
-                lambda monster, qubit: print(
-                    f'Sample result {monster.sample(qubit, False)}'),
+                lambda monster, qubit:
+                f'Sample result {monster.sample(monster.quantum_object_name(qubit), False)}',
```

Comment:
Why remove the `print`?

Reply:
Primarily to make it testable.  That way it returns the string and the caller can do what it wants with it.  I suppose in the future we could also display it in a different way (like a javascript based window or something), but I would imagine that's pretty far off.

Reply:
Somewhat nitpicky but if the caller is expected to do what they want with the sample output, maybe the string message should also be controlled by the caller. `s` can just return the sampled value similar to `m`.

Reply:
I am not sure this is a good idea.  For the example of a sample vs measure, the caller needs to understand the difference between them then, since they are both returning a measurement result, so the caller needs to understand the semantic difference between 'm' and 's' which should really be the logic within the class.

I am going to skip this comment for now.  I have a suspicion we will need to revisit this logic anyway once things progress.